Melhorias em Tiendanube (#2168)
mercadopago/devsite-docs: guides/nuvemshop/prerequisites.en.md

# Prerequisites

In order to integrate Mercado Pago with Tiendanube, you must meet the requirements below.

> WARNING
>
> Important
>

> > If the integrator does not have access to the Mercado Pago account that will be linked to the store, another person with access should carry out the process. To learn how to change the linked Mercado Pago account, refer to the "Change Mercado Pago Account" section in the [Integration configuration documentation.](/developers/en/docs/nuvemshop/integration)

Reviewer: ```suggestion
> If you don't have access to the Mercado Pago account that will be linked to the store, another person with access should carry out the process. To learn how to change the linked Mercado Pago account, refer to the "Change Mercado Pago Account" section in the [Integration configuration documentation.](/developers/en/docs/nuvemshop/integration)
```
